notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>Out of curiosity, which TPU quota does this part end up using? There's a separate quota for Cloud TPUs used via AI Platform, and Cloud TPUs just used directly with Compute Engine, and I don't think most people realize that. Might be a good idea to link people to the [AI Platform Training Quotas page](https://cloud.google.com/ai-platform/training/docs/quotas#tpu-quota) in case someone tries to run this and doesn't have enough quota.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Ah, since it's not going through AI Platform Training at all, the right quotas page to use is this: https://cloud.google.com/tpu/docs/quota</p><p> </p><p>My mistake for not understanding that before</p>